consensus/state_sync/module.go

```diff
@@ -22,14 +29,22 @@ var (
 )
 
 type stateSync struct {
-	bus    modules.Bus
-	logger *modules.Logger
+	bus                    modules.Bus
+	logger                 *modules.Logger
+	validators             []*coreTypes.Actor
```

Comment:
This changes on every height. Is it really the right thing to do it to store it locally rather than reading from the persistence module?

Remember: less state -> better